config/200-clusterrole.yaml

```diff
@@ -74,6 +78,15 @@ rules:
   - apiGroups: ["apiextensions.k8s.io"]
     resources: ["customresourcedefinitions", "customresourcedefinitions/status"]
     verbs: ["get", "list", "update", "patch", "watch"]
```

Comment:
Do you know of a situation where we patch or update our CRD objects?

Reply:
Update is done in knative. If we're already allowing updates, patching seems reasonable as well - https://github.com/tektoncd/pipeline/blob/1f5980f8c8a05b106687cfa3e5b3193c213cb66e/vendor/knative.dev/pkg/webhook/resourcesemantics/conversion/reconciler.go#L111

list/watch are also needed by knative for the informers, and unfortunately also need to be global scope :( https://github.com/tektoncd/pipeline/blob/17439c01768f734468ba1ed69a333238615e568b/vendor/knative.dev/pkg/client/injection/apiextensions/informers/apiextensions/v1/customresourcedefinition/customresourcedefinition.go#L93

Reply:
Nice, thanks for looking into that.